content/_data/events/guadalajara-jam.adoc

```diff
@@ -0,0 +1,5 @@
+---
+title: "Guadalajara JAM"
+date: "2016-04-30T16:00:00"
+link: "http://www.meetup.com/Guadalajara-Jenkins-Area-Meetup/events/230208035/"
+---
```

Comment:
they didn't have much except "Content to come soon"..it was better not to
include that in the description. I'm checking w them to see what stored and
will update.

On Thu, Apr 21, 2016 at 7:21 AM, R. Tyler Croy <notifications@github.com>
wrote:

> In content/_data/events/guadalajara-jam.adoc
> <https://github.com/jenkins-infra/jenkins.io/pull/232#discussion_r60588520>
> :
>
> > @@ -0,0 +1,5 @@
> > +---
> > +title: "Guadalajara JAM"
> > +date: "2016-04-30T16:00:00"
> > +link: "http://www.meetup.com/Guadalajara-Jenkins-Area-Meetup/events/230208035/"
> > +---
>
> Can you add a little text description so the tile on the site looks decent?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/232/files/8f01e4bcd3b20689e64865ef8e6012d726210029#r60588520>
>